SECURITY.md

```diff
@@ -49,16 +88,18 @@ LoopBack implements the
 
 </details>
 
-Within LoopBack 4, fixes are not backported across semver major versions of a
-package.
+**Within LoopBack 4, fixes are not backported across semver major versions of a
+package.**
+
+We don't provide any
+[LTS version for LoopBack 4](https://github.com/strongloop/loopback-next/issues/4398)
+yet.
```

Comment:
I would like to reframe these to paragraph in a more positive way. How about this?

```suggestion
**LoopBack 4 does not provide any Long Term Support version yet, fixes are not back-ported across semver major versions of a package.**

Please join the discussion in [loopback-next#4398](https://github.com/strongloop/loopback-next/issues/4398) if you are interested in an LTS version.
```